core/src/main/java/com/my/core/domain/JourneyPlannerResultDomainModel.kt

```diff
@@ -12,24 +13,24 @@ sealed class JourneyPlannerResultDomainModel {
             val from: String,
             val to: String,
             val via: String
-        )
+        ) : Serializable
 
         data class Journey(
             val arrivalDateTime: LocalDateTime,
             val duration: Int,
```

Comment:
is the duration not inferred from the depart and arrival times?

Reply:
Removed. Now inferred:
        textViewDuration.text = context.getString(
            R.string.textView_duration,
            Duration.between(journey.startDateTime, journey.arrivalDateTime).toMinutes().toString()
        )